src/otg_fs.rs

```diff
@@ -0,0 +1,53 @@
+//! USB OTG full-speed peripheral
+//!
+//! Requires the `usb_fs` feature.
+//! Only one of the `usb_fs`/`usb_hs` features can be selected at the same time.
+
+use crate::pac;
+
+use crate::gpio::{
+    gpioa::{PA11, PA12},
+    Alternate, AF10,
+};
+use crate::time::Hertz;
+
+pub use synopsys_usb_otg::UsbBus;
+use synopsys_usb_otg::UsbPeripheral;
+
+pub struct USB {
```

Comment:
Other peripherals, like Spi and Serial also provide a `new` constructor, IMO it would be more consistent to also add one for USB.

Reply:
Good idea, I've added the constuctor.